Lombiq.Tests.UI/Extensions/NavigationUITestContextExtensions.cs

```diff
@@ -204,26 +213,51 @@ public static Task<Uri> GoToSetupPageAndSetupOrchardCoreAsync(this UITestContext
 
     public static async Task<Uri> GoToSetupPageAndSetupOrchardCoreAsync(
```

Comment:
Keep this method, because it's Page class-specific, and add a new alternative instead (like `GoToSetupAndSetupOrchardCoreAsync`).